source/develop/release-management/features/storage/cd-management.md

```diff
@@ -0,0 +1,117 @@
+---
+title: CD-ROM Management
+category: feature
+authors: vjuranek, nsoffer
+feature_name: CD-ROM Management
+feature_modules: engine,vdsm
+feature_status: Design
+---
+
+# CD-ROM Management
+
+
+## Summary
+
+This feature adds new functions for CD-ROM management, namely for loading, ejecting and changing CD-ROM.
+
+## Owner
+
+- Vojtech Juranek (<vjuranek@redhat.com>)
+
+## Overview
+
+### What was available before this feature?
+
+Before this, feature oVirt engine provided functions for changing and ejecting CD-ROM.
+However, non of these functions worked properly.
+Changing CD failed on block storage as the appropriate volume wasn't activated.
+Ejecting CD failed to deactivate the volume.
+For more details, see following bugs.
+
+### Bugs
+
+- [Error changing CD for a running VM when ISO image is on a block domain](https://bugzilla.redhat.com/1589763)
+- [Volume not deactivated after ejecting CDROM](https://bugzilla.redhat.com/1868643)
+
+
+### Related links
+
+- [Discussion in ovirt devel mailing list.](https://www.mail-archive.com/devel@ovirt.org/msg16082.html)
+- [Vdsm tries to tear down in-use volume of ISO in block storage domain](https://bugzilla.redhat.com/1725915)
+
+
+### Functionality added with this feature
+
+This feature propose new functions to both engine and vdsm for loading, ejecting and changing CD.
+With these functions, all aforementioned functionality will work properly.
+
+#### Vdsm
+
+- Add VM.insertCD API
+- Add VM.ejectCD API
+- Updated recovery flow to check `in_progress` flag for CDs
+
+#### Engine
+
+- Add InsertCD sub-command
+- Add EjectCD sub-command
+- Update ChangeCD to use InsertCD and EjectCD on 4.5 cluster version
+
+
+### Flows
+
+#### VM with empty tray, insert CD
+
+1. engine sends `insertCD` command, specifying ISO by PDIV format
+1. add new drivespec to VM metadata with new attribute `in_progress`
+```
+    <ovirt-vm:device devtype="disk" name="sdc" in_progress="yes">
+```
```

Comment:
indentation not needed when using ``` style. If you want to indent you can remove the back-quotes.

Reply:
for some reason without ``` it doesn't work (at least on my local instance). So ``` are there for proper rendering as HTML and indentation is for reading as a plain text